config/default/GUIDE.md

```diff
@@ -1,14 +1,14 @@
-# Reference configuration changes cookbook
+# Configuring Reference Applications
```

Comment:
It is very specifically and intentionally a "cookbook". Ie it is a collection of "recipes": specific actions to solving specific problems.

This distinguishes it from a general guide / api documentation that is more thorough.

Reply:
Makes sense, I think the intention was to change the "configuration" language more than the "cookbook" portion. @CescaGarrett feel free to comment if I am mistaken.

```suggestion
# Cookbook for Reference Applications
```